OpenMP/Intro.md

```diff
@@ -0,0 +1,28 @@
+# OpenMP with ROCm
+
+## What is OpenMP?
+[OpenMP (Open Multi-Processing)](https://www.openmp.org/) is a "collection of compiler directives, library routines, and environment variables ... for parallelism in C, C++ and Fortran programs [[1](#References)]."
+
+## Why use OpenMP rather than HIP?
+While HIP can only take advantage of parallelism on GPUs, OpenMP is portable and can be used on more than GPUs such as CPUs.
```

Comment:
“Can be used on more than GPUs such as CPUs” is awkward. Try “can work on other hardware types, such as CPUs”